docs/man_pages/general/feature-usage-tracking-status.md

```diff
@@ -0,0 +1,21 @@
+feature-usage-tracking status
+==========
+
+Usage | Synopsis
+------|-------
+General | `$ appbuilder feature-usage-tracking status`
+
+Shows the current configuration for anonymous usage tracking for the Telerik AppBuilder CLI.
```

Comment:
Actually, usage tracking commands set the state for all online clients, including Web Client and Desktop client.